dataprep/eda/correlation/compute/overview.py

```diff
@@ -196,14 +232,67 @@ def _spearman_nxn(df: DataArray) -> da.Array:
 
 
 def _kendall_tau_nxn(df: DataArray) -> da.Array:
-    """Calculate column-wise kendalltau correlation."""
+    """
+    Calculate column-wise kendalltau correlation.
+    """
     return (
         df.frame.repartition(npartitions=1)
         .map_partitions(partial(pd.DataFrame.corr, method="kendall"))
         .to_dask_array()
     )
 
 
+def _cramerv_nxn(df: DataArray) -> da.Array:
```

Comment:
What's the performance / complexity of this function? Is cramerv disabled by default?

Reply:
Haven't test the performance. It is enabled by default. I think it's better to have both categorical correlation and numerical correlation for the overview (we may just choose one numerical correlation such as person correlation for efficiency). Or another choice is to enable Cramerv for some data (e.g., when number of columns is small).  Cramerv should be expensive, around `n^2*v^2`, where `n` is the number of columns and `v` is the distinct values.  The code of this part is similar to pandas-profiling.

Reply:
how about have a try on the titanic and if it is very slow then we disable it by default? We can also discuss in the meeting if we can have the number before the meeting.